### DISCUSSION TITLE
Literartur: Autor und Jahr und Fettschrift

### DISCUSSION THREAD

**Discussion by bdoppels** [andygrunwald/FOM-LaTeX-Template#214] [Q&A]
Halli Hallo,

ich zweifele gerade daran, den Autoren und das Jahr in fettschrift zu schreiben, scheiter aber vergeblich.

Ich verwende die modsBiblatex2018

LG

**Comment by andygrunwald** [comment]
Hey @bdoppels,

meinst du das LiteraturVerzeichnis? Oder welche Stelle sprichst du an?

**Reply by bdoppels** [reply]
Hey @andygrunwald 
genau!

**Reply by bdoppels** [reply]
hat sich geklärt, falls das von interesse ist:

https://stackoverflow.com/questions/67575574/biblatex-customizing-bibliography-entry/67586664#67586664

**Reply by andygrunwald** [reply]
Cool. Danke fürs follow up!
